pkg/kubelet/kuberuntime/helpers_linux.go

```diff
@@ -49,7 +49,7 @@ func milliCPUToShares(milliCPU int64) int64 {
 }
 
 // milliCPUToQuota converts milliCPU to CFS quota and period values
-func milliCPUToQuota(milliCPU int64, period int64) (quota int64) {
+func milliCPUToQuota(milliCPU int64, period int64) (quota int64, p int64) {
```

Comment:
IMO both inputting and returning the period seems strange.
Could we instead move the feature gate check to `kuberuntime_container_linux.go`, and pass the _actual_ period we are using to this function?

I.E.
```
if utilfeature.DefaultFeatureGate.Enabled(kubefeatures.CPUCFSQuotaPeriod) {
    period = int64(m.cpuCFSQuotaPeriod.Duration / time.Microsecond)
} else {
    period = quotaPeriod
}
```

That way it is clear what the feature gate changes, and we avoid confusion from "set to X, and then reset to Y" confusion.

Reply:
Thanks for the suggestion, makes much more sense.
I pushed a fix for the commented code.